<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThreadsDlg</class>
 <widget class="QDialog" name="ThreadsDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>444</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>704</width>
    <height>444</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>704</width>
    <height>444</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ThreadsDlg</string>
  </property>
  <widget class="QGroupBox" name="groupBoxStarting">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>261</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Input Starting Data:</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="labelTotalThreads">
       <property name="text">
        <string>Total Threads of Execution:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxExecution">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>261</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Execution</string>
   </property>
   <widget class="QPushButton" name="pushButtonStart">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxResult">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>681</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Result</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>50</y>
      <width>651</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLCDNumber" name="lcdNumberResult">
       <property name="digitCount">
        <number>10</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonExit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>390</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
